app/views/layouts/application.html.slim

```diff
@@ -105,11 +129,12 @@ html
         / FIXME: This provides alternate modal markup during the Bootstrap 4 migration.
         / When Bootstrap 4 becomes the default, this should also become the default.
         - if content_for?(:head_style_tags) || params.has_key?(:bootstrap4)
-          .single-panel--wrapper
-            .single-panel--medium.md-panel
-              .md-content
-              = link_to "#", class: "md-close js-deny"
-                = render "application/icon_x"
+          .single-panel--wrapper.single-panel--wrapper--collapsed
```

Comment:
At some point the style were refactored but this modal was not cleaned up. This fixed the modals to have proper padding and removes the minimum height (which seems odd on a modal).